lib/rummager_finder_publisher.rb

```diff
@@ -31,6 +31,9 @@ def pre_production?(schema)
 
   def export_finder(schema)
     presenter = FinderRummagerPresenter.new(schema[:file], schema[:timestamp])
-    RummagerWorker.perform_async(presenter.type, presenter.id, presenter.to_json)
+
+    logger.info("Publishing '#{schema[:file]['name']}' finder")
+
+    Services.rummager.add_document(presenter.type, presenter.id, presenter.to_json)
```

Comment:
Yep definitely - most of the tasks in whitehall work that way because there's a massive amount of content to send, but this one is very small.